articles/virtual-machines/dedicated-hosts-how-to.md

```diff
@@ -64,6 +71,18 @@ Not all host SKUs are available in all regions, and availability zones. You can
 az vm list-skus -l eastus2  -r hostGroups/hosts  -o table
 ```
 
+You can also verify if a VM SKU supports Ultra disks.
```

Comment:
VM series or VM family, SKU isn't used much in relation to VM.

Reply:
Got it, fixed it as VM families